src/audiences/edit/index.js

```diff
@@ -0,0 +1,235 @@
+				<div className="error msg">
+					<p>{ error.message }</p>
+				</div>
+			) }
+
+			{ AudienceOptionsUI && ReactDOM.createPortal( <Options audience={ audience } />, AudienceOptionsUI ) }
+
+			<div className="audience-editor__include">
+				<SelectInclude
+					onChange={ e => updateAudience( { include: e.target.value } ) }
+					value={ audience.include }
+					name="audience[include]"
+					label={ __( 'groups', 'altis-analytics' ) }
+				/>
+			</div>
+
+			{ audience.groups.map( ( group, groupId ) => {
+				return (
+					<Group key={ groupId }>
+						<div className="audience-editor__group-header">
+							<h3>{ __( 'Group', 'altis-analytics' ) } { groupId + 1 }</h3>
+							<SelectInclude
+								onChange={ e => updateGroup( groupId, { include: e.target.value } ) }
+								value={ group.include }
+								name={ `audience[groups][${ groupId }][include]` }
+								label={ __( 'rules', 'altis-analytics' ) }
+							/>
+						</div>
+						{ group.rules.map( ( rule, ruleId ) => {
+							const currentField = fields.filter( fieldData => fieldData.name === rule.field )[0] ?? {};
+
+							return (
+								<Rule key={ ruleId }>
+									<select
+										className="audience-editor__rule-field"
+										onChange={ e => updateRule( groupId, ruleId, { field: e.target.value } ) }
+										value={ rule.field }
+										name={`audience[groups][${groupId}][rules][${ruleId}][field]`}
+									>
+										<option value="" className="placeholder">{ __( 'Select a field', 'altis-analytics' ) }</option>
+										{ fields.map( field => (
+											<option key={ field.name } value={ field.name }>{ field.label }</option>
+										) ) }
+									</select>
+									<select
+										className="audience-editor__rule-operator"
+										onChange={ e => updateRule( groupId, ruleId, { operator: e.target.value } ) }
+										value={ rule.operator }
+										name={`audience[groups][${groupId}][rules][${ruleId}][operator]`}
+									>
+										{ ! currentField.type || currentField.type === 'string' && (
+											<Fragment>
+												<option value="=">{ __( 'equals', 'altis-analytics' ) }</option>
+												<option value="!=">{ __( 'does not equal', 'altis-analytics' ) }</option>
+												<option value="*=">{ __( 'contains', 'altis-analytics' ) }</option>
+												<option value="!*">{ __( 'does not contain', 'altis-analytics' ) }</option>
+												<option value="^=">{ __( 'begins with', 'altis-analytics' ) }</option>
+											</Fragment>
+										) }
+										{ currentField.type === 'number' && (
+											<Fragment>
+												<option value="gt">{ __( 'is greater than', 'altis-analytics' ) }</option>
+												<option value="lt">{ __( 'is less than', 'altis-analytics' ) }</option>
+												<option value="gte">{ __( 'is greater than or equal to', 'altis-analytics' ) }</option>
+												<option value="lte">{ __( 'is less than or equal to', 'altis-analytics' ) }</option>
+											</Fragment>
```

Comment:
Could these maybe be factored into constants of `'op' => 'description'` for readability?